# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/rsynth/rsynth-use-devfsdsp.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./config/linuxplay.c.orig	2003-07-25 11:47:04.710496048 +0200
+++ ./config/linuxplay.c	2003-07-25 11:29:16.000000000 +0200
@@ -38,7 +38,7 @@
 
 static int dev_fd = -1;
  /* file descriptor for audio device */
-char *dev_file = "/dev/sbdsp";
+char *dev_file = "/dev/sound/dsp";
 
 static int linear_fd = -1;
 
